Documentation/Storage-Configuration/ceph-teardown.md

```diff
@@ -2,6 +2,23 @@
 title: Cleanup
 ---
 
+## Cleaning up a Custom Resource
+
+To cleanup a specific custom resource, add `ceph.rook.io/force-deletion="true"` annotation before deleting it. Rook will start a cleanup job that will delete the all the ceph resources created by that custom resource. For example, run the following commands to clean the `CephFSSubVolumeGroup` resource named `myfs-csi`:
+
+``` console
+kubectl annotate cephfilesystemsubvolumegroups.ceph.rook.io myfs-csi -n rook-ceph ceph.rook.io/force-deletion="true"
+kubectl delete cephfilesystemsubvolumegroups.ceph.rook.io myfs-csi -n rook-ceph
```

Comment:
For consistency in the docs, let's put the namespace first. We aren't very consistent about it, but IMO it is a nice convention.
```suggestion
kubectl -n rook-ceph delete cephfilesystemsubvolumegroups.ceph.rook.io my-subvolumegroup
```